data/api/tmdb/src/main/java/io/filmtime/data/api/tmdb/CreditsExt.kt

```diff
@@ -0,0 +1,13 @@
+package io.filmtime.data.api.tmdb
+
+import io.filmtime.data.model.CreditItem
+import io.filmtime.data.network.Cast
+
+private val TMDB_BASE_IMAGE_URL = "https://image.tmdb.org/t/p/original"
```

Comment:
Please do not duplicate `TMDB_BASE_IMAGE_URL`, you can make the one in `VideoThumbnailExt.kt` internal and reuse here, so we have only one place to modify this base url in future